packages/web/components/Landing/Game.tsx

```diff
@@ -1,25 +1,83 @@
-import { Text, VStack } from '@metafam/ds';
+import { Box, Button, Container, Text } from '@metafam/ds';
 import BackgroundImage from 'assets/landing/game-background.png';
 import { FullPageContainer } from 'components/Container';
 import { MetaLink } from 'components/Link';
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import { useRouter } from 'next/router';
+import { useRef } from 'react';
+import { BsArrowDown } from 'react-icons/bs';
 
-export const Game: React.FC = () => (
-  <FullPageContainer bgImageUrl={BackgroundImage}>
-    <VStack
-      align="stretch"
-      fontSize={{ base: 'xl', md: '5xl' }}
-      color="white"
-      maxWidth={{ base: '16rem', md: '32rem' }}
+export const Game: React.FC = () => {
+  const ref = useRef<HTMLDivElement>(null);
+  const onScreen = useOnScreen(ref);
+  const router = useRouter();
+  const section = 'section-2';
+  const handleSectionNav = (sectionId: string) => {
+    router.push(`#${sectionId}`);
+  };
+
+  return (
+    <FullPageContainer
+      bgImageUrl={BackgroundImage}
+      id={section}
+      position="relative"
     >
-      <Text>
-        “Metagame is any approach to a game that transcends or operates outside
-        of the prescribed rules of the game, uses external factors to affect the
-        game, or goes beyond the supposed limits or environment set by the
-        game.”
-      </Text>
-      <Text textAlign="right">
-        - From <MetaLink href="#">The Wiki</MetaLink>
-      </Text>
-    </VStack>
-  </FullPageContainer>
-);
+      <Container
+        d="flex"
+        maxW={{ base: '100%', md: '7xl', '2xl': '8xl' }}
+        height="100%"
+        alignItems="center"
+      >
+        <Box
+          ref={ref}
+          display="flex"
+          flexDirection="column"
+          justifyContent="center"
+          maxWidth="2xl"
+          fontSize={{ base: '1.5rem', md: '2.375rem' }}
+          lineHeight={{ base: '2.35rem', md: '3.5rem' }}
+          pl={{ base: 0, md: 0 }}
+          zIndex={100}
+          transform={`translate3d(0, ${onScreen ? '0' : '50px'}, 0)`}
+          opacity={onScreen ? 1 : 0}
+          transition="transform 0.3s 0.1s ease-in-out, opacity 0.5s 0.2s ease-in"
+        >
```

Comment:
looks like this pattern is repeating itself throughout the pages, could you make a reusable component out of it? it's gonna make it much easier easier to maintain

Reply:
of course. I wasn't asking for a review yet, hence the draft pr. I just wanted to get it up on preview for feedback on the ui. 

I get heavily slowed down by typescript and general programming and find it much faster to work like this initially, see the repetition, try and split everything out and then seek support from more experienced devs when i inevitably fail. 😄

Reply:
ok, pardon for the premature review :P also i work in a similar way, just wanted to point out something that i've noticed 👌